website/docs/fabric.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 5
+title: "Microsoft Fabric"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Querying from Microsoft Fabric
+This guide offers a short tutorial on how to query Apache Iceberg and Apache Hudi tables in Microsoft Fabric utilizing 
+the translation capabilities of OneTable. This tutorial is intended solely for demonstration and to verify the 
+compatibility of OneTable's output with Fabric. The tutorial leverages the currently[^1] available features in Fabric, like 
+`Shortcuts`.
+
+
+## What is Microsoft Fabric
+Microsoft Fabric is a unified data analytics platform for data analytics. It offers a wide range of capabilities 
+including data engineering, data science, real-time analytics, and business intelligence. At its core is the data 
+lake, OneLake, which provides tabular data stored in Delta Parquet format, allowing one copy of the data to be used in 
+all Fabric analytical services like T-SQL, 
+Spark, and Power BI. OneLake is designed to do away with the need for data duplication and eliminates the necessity 
+for data migration or transfers. Whether it's a data engineer adding data to a table using Spark or a SQL developer 
+analyzing data with T-SQL in a data warehouse, both can access and work on the same data copy stored in OneLake. 
+Additionally, OneLake enables integration of existing storage accounts via its 'Shortcut' feature, which acts as a link
+to data in other file systems.
+
+## Tutorial
+The objective of the following tutorial is to translate an Iceberg or Hudi table in ADLS storage account into Delta Lake
+format using OneTable. After translation, this table will be accessible for querying from various Fabric engines,
+including T-SQL, Spark, and Power BI.
+
+### Pre-requisites
+* An active Microsoft Fabric workspace ([ref](https://learn.microsoft.com/en-us/fabric/get-started/workspaces)).
```

Comment:
Can we do like [Microsoft Fabric workspace](https://learn.microsoft.com/en-us/fabric/get-started/workspaces) instead? Just to stay consistent with other pages.